mkosi/resources/man/mkosi.1.md

```diff
@@ -2965,6 +2965,15 @@ Note that the minimum required Python version is 3.9.
   Alternatively, the `z` or `Z` directives for `systemd-tmpfiles` can be used to chown
   various directories and files to their owning user when the system first boots up.
 
+- Why does `portablectl inspect` say my portable service isn't one?
```

Comment:
did you mean dissect? I am pretty sure portablectl works just fine

Reply:
Yes it is a dissect problem.

Portablectl does not report anything weird (or much at all, which is why I've beeb using dissect).